src/platforms/mirserver/mirserverhooks.cpp

```diff
@@ -93,6 +115,7 @@ struct qtmir::MirServerHooks::Self
     std::weak_ptr<mir::shell::DisplayConfigurationController> m_mirDisplayConfigurationController;
     std::weak_ptr<mir::scene::PromptSessionManager> m_mirPromptSessionManager;
     std::weak_ptr<mir::input::InputDeviceHub> m_inputDeviceHub;
+    CreateNamedCursor create_cursor;    
```

Comment:
I think the QtMir naming style would have this as `m_createCursor`. But the style around here is inconsistent anyway.